.travis.yml

```diff
@@ -19,6 +19,7 @@ before_script:
   - python manage.py collectstatic --noinput -v 0
   - python manage.py update_product_details
   - svn checkout https://svn.mozilla.org/projects/mozilla.com/trunk/locales/ locale
+  - python scripts/check_calendars.py
```

Comment:
I'd prefer this go above the `mysql` line just so it fails more quickly before all that other work is done.